packages/@orbit/coordinator/test/index.ts

```diff
@@ -0,0 +1,8 @@
+// DO NOT EDIT - This is an auto-generated index of tests and will be overwritten.
```

Comment:
autogenerated files are still committed?

Reply:
Good call, I should git ignore those.